### PR TITLE
Feat: Added input and image component

### PR DESCRIPTION
**Pull request by Richa-rani02**
Added below input boxes:

1.simple input box
2.label input box
3.disabled input box
4.input box with required validation
5.input box with error

Added below images
1.responsive image
2.image with round corner
3.rounded image
demoinput

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
looks good

File: components/images.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+
+.img-responsive{
+    object-fit: contain;
+    width: 80%;
```
Comment (Hardiegogo) [inline components/images.css]: For a responsive image, you should give `width:100%`. Use this image inside a div and set the width and height property and the image will adapt.

**Review by Hardiegogo — COMMENTED** [review]
Looks good. You can merge this after making some changes to your responsive image properties.

File: components/input.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.input-container{
+  width: 100%;
+  margin: 0.5rem 1rem 0 1rem;
+  }
+  .input-label-container{
+      margin: 0.5rem 1rem 0 1rem;
```
Comment (nerdyspook) [inline components/input.css:6]: You can define global variables for spacing also to keep it consistent

File: components/images.css (hunk 1 above)
Comment (Richa-rani02) [reply components/images.css]: Required changes done

File: components/input.css (hunk 2 above)
Comment (Richa-rani02) [reply components/input.css:6]: yes will do this when refactoring code

File: components/images.css [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 .img-responsive{
     object-fit: contain;
-    width: 80%;
+    width: 100%;
     height: 30%;
```
Comment (Hardiegogo) [inline components/images.css:6]: `height: auto` will work better here, as the image will maintain its aspect ratio.

Comment (Hardiegogo) [reply components/images.css:6]: Good work Richa.

**Pull request merged**